node.gypi

```diff
@@ -301,6 +301,9 @@
       'ldflags': [ '-Wl,-z,relro',
                    '-Wl,-z,now' ]
     }],
+    [ 'OS=="linux" and target_arch=="x64" and node_use_large_pages=="true"', {
+      'ldflags': [ '-Wl,-T <(PRODUCT_DIR)/../../src/large_pages/ld.implicit.script' ]
```

Comment:
```suggestion
      'ldflags': [
        '-Wl,-T',
        '<!(realpath src/large_pages/ld.implicit.script)',
      ]
```
This should work, and be cross platform

Reply:
@refack Thanks for the suggestion. I'll make the change and update the PR.